packages/alfa-style/test/property/clip.spec.tsx

```diff
@@ -21,8 +21,8 @@ test("#cascaded() parses `clip: auto`", (t) => {
   });
 });
 
-test("#cascaded() parses `clip: rect(1px, auto, 2em, auto)`", (t) => {
-  const element = <div style={{ clip: "rect(1px, auto, 2em, auto)" }} />;
+test("#cascaded() parses `clip: rect(1px auto 2em auto)`", (t) => {
+  const element = <div style={{ clip: "rect(1px auto 2em auto)" }} />;
```

Comment:
That test case shouldn't change. It's important to keep the dual syntax (space / comma) and check that both are parsed.
```suggestion
test("#cascaded() parses `clip: rect(1px, auto, 2em, auto)`", (t) => {
  const element = <div style={{ clip: "rect(1px, auto, 2em, auto)" }} />;
```